internal/sql/repository/pipelineConfig/CdWorfkflowRepository.go

```diff
@@ -591,14 +614,16 @@ func (impl *CdWorkflowRepositoryImpl) FetchArtifactsByCdPipelineId(pipelineId in
 
 func (impl *CdWorkflowRepositoryImpl) GetLatestTriggersOfHelmPipelinesStuckInNonTerminalStatuses(getPipelineDeployedWithinHours int) ([]*CdWorkflowRunner, error) {
 	var wfrList []*CdWorkflowRunner
+	excludedStatusList := WfrTerminalStatusList
+	excludedStatusList = append(excludedStatusList, WorkflowInitiated, WorkflowInQueue)
```

Comment:
we should also add Starting state in this list

Reply:
updated 👍